.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
since this is dependant on your environment, ill put this as a global .gitignore in your local setup.